torch/_dynamo/output_graph.py

```diff
@@ -511,10 +515,14 @@ def register_attr_or_module(
         assert not isinstance(source, ParamBufferSource)
 
         if isinstance(target, torch.Tensor):
-            if len(self.tracers) > 1:
-                unimplemented(
-                    "accessing attribute of nn.Module inside HigherOrderOperator"
-                )
+            tracer = self.current_tracer
+            if not self.is_root_tracer():
```

Comment:
Can this handle nested HigherOrderOperator calls?

Reply:
Good point. Maybe not.

Reply:
Can you give me an example on how to test this? I think it should work. We will still raise the params/buffers as inputs for all the inner graphs, and the outermost_graph will make the right connection.

Reply:
Yeah i was just thinking about nested control flow